src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -55,6 +58,11 @@ export class BaseEditCourseForm extends React.Component {
       updateFormValuesAfterSave,
     } = this.props;
 
+    if (initialCourseRuns.length && !this.state.collapsiblesOpen.length) {
+      const arr = initialCourseRuns.map(() => (false));
```

Comment:
Nit: name this variable something more helpful. Maybe just calling it `const collapsiblesOpen =`